exasol/ds/sandbox/lib/dss_docker/create_image.py

```diff
@@ -162,7 +168,6 @@ def _commit_container(
         notebook_folder_initial = get_fact(facts, "notebook_folder", "initial")
         conf = {
             "Entrypoint": entrypoint(facts),
-            "User": get_fact(facts, "jupyter", "user"),
```

Comment:
Should we use Ubuntu as the default user and start the entrypoint with sudo

Reply:
I will add the Ansible installer user `ubuntu` as Ansible fact, and use it for running the Docker container.

Reply:
see latest push